samples/templates/default-azuredeploy-docker.json

```diff
@@ -154,7 +154,7 @@
         "sqlSchemaAutomaticUpdatesEnabled": {
             "type": "string",
             "allowedValues":["auto", "tool"],
-            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), 'tool', '')]",
+            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), 'tool', 'auto')]",
```

Comment:
We are intentionally leaving this field blank. Both the tool and auto settings have drawbacks and we don't want to put one as the default. Leaving it blank forces users to look at the description of the field and make a conscious choice for which they want.

Reply:
Got it! I'll revert this and add a sqlSchemaAutomaticUpdatesEnabled=auto argument to the sample SQL quickstart commands

Reply:
Just added that reversion and manual testing for the new PS and AzCLI quickstarts looks good, so marking this as closed. Please let me know if you have any other feedback on this!